### PR TITLE
Y Dwarf Pm Ingest 

### PR DESCRIPTION
**Pull request by ebhort27**
Proper motion ingest scripts for Y Dwarf

### CODE REVIEW

File: scripts/ingests/Y_dwarf_pm_ingest.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+from astropy.table import Table
+import numpy as np
+from astropy.coordinates import SkyCoord
+import astropy.units as u
+from astroquery.simbad import Simbad
+import warnings
+warnings.filterwarnings("ignore", module='astroquery.simbad')
+import re
+import os
+from pathlib import Path
```
Comment (kelle) [inline scripts/ingests/Y_dwarf_pm_ingest.py:19]: Many of these things are not required. In the future, check to be sure you only import what you use.

File: tests/test_data.py [hunk 2]
```diff
@@ -124,7 +127,12 @@ def test_Kirk19_ingest(db):
...
-    # Test parallaxes added
+    # Test parallaxes 
     ref = 'Kirk19'
     t = db.query(db.Parallaxes).filter(db.Parallaxes.c.reference == ref).astropy()
     assert len(t) == 22, f'found {len(t)} parallax entries for {ref}'
+    
+    #Test proper motions added
+    ref = 'Kirk19'
+    t = db.query(db.ProperMotions).filter(db.ProperMotions.c.reference == ref).astropy()
+    assert len(t) == 22, f'found {len(t)} proper motion entries for {ref}'
```
Comment (kelle) [inline tests/test_data.py:138]: Add new line
```suggestion
    assert len(t) == 22, f'found {len(t)} proper motion entries for {ref}'
    
```

**Review by kelle — APPROVED** [review]
Even though the ingest script could be better, the utils scripts look good for now and most importantly, the new json files look correct.

**Comment by kelle** [comment]
@dr-rodriguez , could you take a quick look before we merge?

**Review by dr-rodriguez — APPROVED** [review]
Look good. The addition to test_data is perfect.

**Pull request merged**